### PR TITLE
Fixed #30486 -- Updated custom Count aggregate example to use allow_distinct.

### PR DESCRIPTION
**Pull request by VVyacheslav**

### CODE REVIEW

File: docs/ref/models/expressions.txt [hunk 1]
```diff
@@ -239,7 +239,7 @@ For example, to sort companies that haven't been contacted (``last_contacted``
 is null) after companies that have been contacted::
 
     from django.db.models import F
-    Company.object.order_by(F('last_contacted').desc(nulls_last=True))
+    Company.objects.order_by(F('last_contacted').desc(nulls_last=True))
```
[the remark this answers is not in the document]
Comment (carltongibson) [reply docs/ref/models/expressions.txt]: Merged in 60b6a7ea9d9bd759ee623f080165416bc0207e59. (You can rebase this PR.) Thanks!

**Comment by carltongibson** [comment]
I created the ticket: https://code.djangoproject.com/ticket/30486#ticket

**Comment by carltongibson** [comment]
(@VVyacheslav if you like, you can assign the Trac ticket to yourself, to show that you're working on it. 🙂)

**Comment by VVyacheslav** [comment]
> (@VVyacheslav if you like, you can assign the Trac ticket to yourself, to show that you're working on it. 🙂)

Yes, please. Thank you! 🙂

**Pull request closed without merging**